dev-docs/bidders/oprx.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: OptimizeRx
+description: Prebid OptimizeRx Bidder Adapter
+biddercode: oprx
+tcfeu_supported: false
+usp_supported: false
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+userId: none
+media_types: banner
+safeframes_ok: true
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
```

Comment:
```suggestion
pbs: false
```

I would recommend setting this to `false`. Then we can merge this docs PR and change it back to `true` in a followup PR, once prebid server is merged. Prebid Server tends to take a little longer